src/patchCreateProgram.ts

```diff
@@ -1,5 +1,5 @@
 import * as ts from 'typescript'
-import {PluginCreator} from './PluginCreator'
+import {PluginCreator, preparePluginsFromCompilerOptions} from './PluginCreator'
```

Comment:
May be move preparePluginsFromCompilerOptions to patchCreateProgram.ts or incapsulate it into PluginCreator.createTransformers?